components/core/tools/scripts/lib_install/openssl.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+cUsage="Usage: ${BASH_SOURCE[0]} <version> [prefix] [openssldir]"
+if [ "$#" -lt 1 ] ; then
+    echo $cUsage
+    exit
+fi
+version="$1"
+prefix="${2:-/opt/openssl-${version}}"
+openssldir="${3:-$prefix/ssl}"  # Default to prefix/ssl if not specified
+
+package_name=openssl-${version}
+temp_dir="/tmp/${package_name}-installation"
+mkdir -p "$temp_dir"
+
+echo "🔧 OpenSSL version: $version"
+echo "📁 Install prefix: $prefix"
+echo "📂 OpenSSL directory: $openssldir"
+
+echo "Checking for elevated privileges..."
+if [ ${EUID:-$(id -u)} -ne 0 ] ; then
+  sudo echo "Script can elevate privileges."
+fi
+
+# Download and extract
+cd "${temp_dir}"
+curl -LO "https://www.openssl.org/source/openssl-${version}.tar.gz"
+tar -xf "openssl-${version}.tar.gz"
+cd "openssl-${version}"
+
+# Configure and build
+./config --prefix="$prefix" --openssldir="$openssldir" -fPIC
```

Comment:
```suggestion
./config --prefix="${prefix}" --openssldir="${openssldir}"
```
- Adding `{}`
- Let's not add `-fPIC` for now in open source. I know this is required for the private version, but for oss we'd prefer to add it later with proper benchmark.